src/post.ts

```diff
@@ -40,6 +41,9 @@ export async function post (): Promise<void> {
 
     slackChannel = getInput('slack_channel') ?? ''
     console.log(`slack_channel: ${slackChannel}`)
+
+    deploymentConfidenceUrl = getInput('deployment_confidence_url') ?? ''
+    console.log(`deployment confidence dashboard URL: ${deploymentConfidenceUrl}`)
```

Comment:
```suggestion
    deploymentConfidenceUrl = getInput('deployment_confidence_url')
    if (deploymentConfidenceUrl) {
      console.log(`deployment confidence dashboard URL: ${deploymentConfidenceUrl}`)
    }
```

Reply:
Thinking of keeping it as is since it seems like the approach we follow for all the other inputs. Shouldn't cause any issues imo 😊